src/components/Dialogs/Dialogs.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import style from './Dialogs.module.css'
+import { NavLink } from 'react-router-dom';
+
+const DialogItem = (props) => {
+    let path = '/dialogs/' + props.id;
+
+    return (
+        <div className={style.dialog + ' ' + style.active}>
+            <NavLink to={path}>{props.name}</NavLink>
+        </div>
+    );
+};
+
+const Message = (props) => {
+    return (
```

Comment:
Consider using a short form of "arrow functions with immediate return statement"

Reply:
/fxd. And in future versions I will start to use .map